Dockerfiles/Dockerfile

```diff
@@ -1,9 +1,18 @@
+FROM golang:1.15-alpine as myiac-builder
+
+ADD .. /workdir
```

Comment:
@pozgo , I am noticing that for the command `docker build -t my_image -f Dockerfiles/Dockerfile .` to work, I needed to put `ADD .. /workdir` instead of the above. The error I've got is:

```
⚡ % docker build -t my_image -f Dockerfiles/Dockerfile .                                                                             ~/development/repos/myiac
Sending build context to Docker daemon  22.57MB
Step 1/9 : FROM golang:1.15-alpine as myiac-builder
 ---> 1463476d8605
Step 2/9 : ADD .. /workdir
ADD failed: Forbidden path outside the build context: .. ()
```

What can be causing this? maybe Docker version? I am still on 

```
(feature/crema-integration)⚡ % docker -v                                                                                                                       ~/development/repos/moneycol/server
Docker version 19.03.4, build 9013bf5
```

Reply:
Could be the Docker version 

I tested it on 
```
❯ docker -v
Docker version 20.10.2, build 2291f61
```